doc/contributing/commit-queue.md

```diff
@@ -122,3 +120,7 @@ once we start using the Commit Queue we notice that the number of required
 reverts increases drastically, we can pause the queue until a Revert Queue is
 implemented, but until then we can enable the Commit Queue and then work on a
 Revert Queue as a follow-up.
+
+[^1]: A PR is considered mergeable when it's been open for at least two days (unless
+    it's being fast-tracked) and has more than one collaborator's approval or if it's
+    been open for at least seven days and has at least one collaborator's approval.
```

Comment:
PS: I am repeating some info here, I was thinking of adding a link to https://github.com/nodejs/node/blob/7112fb9fd7281917c268959bfb25e8f12f41218e/doc/contributing/pull-requests.md#step-10-landing but that section doesn't contain any info regarding fast-tracking :thinking: 

there's also https://github.com/nodejs/node/blob/5f841fb2c8f192766ce4233d47264125cbaa499c/doc/contributing/collaborator-guide.md#waiting-for-approvals which does mention fast-tracking and the two days rule but does not mention the 7 days rule...

maybe all of this should be consolidated into a single place? :thinking:

Reply:
mh... I figured that maybe it's not really too beneficial going into details here :thinking: